kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java

```diff
@@ -1863,6 +1913,7 @@ private void assertTaskInstace(TaskInstance expected, TaskInstance actual) {
         assertEquals(expected.getProcessId(), actual.getProcessId());
         assertEquals(expected.getContainerId(), actual.getContainerId());
         assertEquals(expected.getProcessInstanceId(), actual.getProcessInstanceId());
+        assertEquals(expected.getSlaCompliance(), actual.getSlaCompliance());
```

Comment:
Just noticed that here we don't compare workitemId, which is quite a crucial part when we want to compare 2 TaskInstances. Do you mind adding it here?